### DISCUSSION TITLE
I cannot use the Lighthouse Deck with Lighthouse V2 base station (in cfclient)

### DISCUSSION THREAD

**Discussion by doltae09** [bitcraze/discussions#596] [Q&A] [answered]
https://www.youtube.com/watch?v=DCEHht72B08
https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/

Following the guide as above, I could see a different state from the guide in the cfclient as shown below.
![Screenshot from 2023-03-01 02-20-03](https://user-images.githubusercontent.com/79627957/221931160-d10d3185-2fa1-4cc9-ae50-030493939052.png)
![Screenshot from 2023-03-01 02-24-16](https://user-images.githubusercontent.com/79627957/221934901-6a191da2-ccdd-421f-ab79-f79e45f7a4aa.png)

I setup Lighthouse V2 base station 2EA in cfclient as follows.
![Screenshot from 2023-03-01 00-54-55](https://user-images.githubusercontent.com/79627957/221931541-964a7347-4e1d-427c-9f21-60b99969697e.png)
![KakaoTalk_20230301_023713302_02](https://user-images.githubusercontent.com/79627957/221934388-a14e3d96-70bd-4280-b68e-9ffb458c9b39.jpg)

![Screenshot from 2023-03-01 00-55-46](https://user-images.githubusercontent.com/79627957/221931505-d00bb465-8925-4758-8ed5-46db06974056.png)
![KakaoTalk_20230301_023713302_01](https://user-images.githubusercontent.com/79627957/221934413-84b02098-96ec-45fd-8f16-88f5b9cdecaa.jpg)

And light statement of Crazyflie 2.1 is right this 
![KakaoTalk_20230301_023713302_03](https://user-images.githubusercontent.com/79627957/221934538-3e0fd995-0c95-48c3-bdc1-0f1390125b6a.jpg)
-> "red" & "blue" from reference video
https://www.youtube.com/watch?v=2EfAjeqUtrI
![Screenshot from 2023-03-01 02-39-51](https://user-images.githubusercontent.com/79627957/221933904-c7b5dc70-0677-4c3b-a17c-89f9ddac4939.png)

Total experiment environment is right this
![KakaoTalk_20230301_023713302](https://user-images.githubusercontent.com/79627957/221934779-be14b6fe-9bd6-4e4f-8877-533446c9829c.jpg)

What is the problem now condition? I need help!

**Comment by krichardsson** [comment, marked as answer]
The reason that it does not work is because the firmware (FPGA binary) in the deck needs to be updated (see the tutorial, step [`Update firmware in the Crazyflie and Lighthouse deck`](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/#update-firmware-in-the-crazyflie-and-lighthouse-deck))

The deck must be mounted on the Crazyflie while flashing the Crazyflie from the client. During the flashing process, the client checks which decks that are mounted and updates them if required.

The way to see that the deck must be updated is to look in the console log

```
LHFL: The deck bitstream does not match the required bitstream.
```

**Reply by doltae09** [reply]
I can solve the problem by your opinion! Thank you! sir!

**Reply by knmcguire** [reply]
@doltae09 could you press 'mark as answer' if you are satisfied with the solution?

**Reply by doltae09** [reply]
yes! I miss the this point haha thank you! sir!
